### PR TITLE
Completed a final read

### PR DESCRIPTION
**Pull request by cgreene**
Lots of small changes - not much large - a bit of reordering.

### CODE REVIEW

File: content/05.main-text.md [hunk 1]
```diff
@@ -111,81 +110,23 @@ or translational distance models [@transe].
...
-Clicking an edge displays its properties, informing the user that association between Creutzfeldt-Jakob disease and _NPAS2_ was detected by GWAS.
-<!--
-  This example is also used at https://slides.com/dhimmel/rocky2019#/4
-  More info at https://github.com/greenelab/connectivity-search-manuscript/issues/7
--->
-](https://github.com/greenelab/connectivity-search-manuscript/raw/b290b4ad435553c7126867e0720f3112b4692809/content/media/webapp/v3/webapp.png){#fig:webapp width="100%" .white}
-
-[alzheimer-metapaths]: https://het.io/search/?source=17287&target=7607&metapaths=DaGiGpPW%2CDdGiGpPW%2CDdGpPW%2CDlAeGpPW%2CDrDaGpPW%2CDrDuGpPW%2CDuGiGpPW&complete=
+Completing hetnet connectivity search involved advances on three fronts.
+We implemented new software for efficient software for matrix-based operations on hetnets.
```
Comment (dhimmel) [inline content/05.main-text.md]:
```suggestion
We implemented new software for efficient matrix-based operations on hetnets.
```

File: content/05.main-text.md [hunk 2]
```diff
@@ -111,81 +110,23 @@ or translational distance models [@transe].
...
+We developed strategies to efficiently calculate an appropriate metric under the null.
```
Comment (dhimmel) [inline content/05.main-text.md]: Double check this sentence.

**Review by dhimmel — COMMENTED** [review]
Thanks @cgreene. See the minor suggested change and double check the noted sentence. Then I'll merge!

File: content/05.main-text.md (hunk 2 above)
Comment (cgreene) [reply content/05.main-text.md]: Can you check the new sentence? I'm trying to avoid saying DWPC, but also to give an idea of the work there.

**Review by dhimmel — APPROVED** [review]

**Pull request merged**